src/test/expand.test.ts

```diff
@@ -119,6 +119,9 @@ describe('Expand Abbreviations', () => {
 	// https://github.com/microsoft/vscode/issues/114923
 	testExpandWithCompletion('html', 'figcaption', '<figcaption>${0}</figcaption>');
 
+	// https://github.com/microsoft/vscode/issues/117154
```

Comment:
Nitpick: can you move this addition to a later part of the document? The tests are currently somewhat sorted by issue id

Reply:
Sure thing!